packages/teams-js/src/public/interfaces.ts

```diff
@@ -368,154 +421,204 @@ export interface Context {
   userObjectId?: string;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.team.isArchived} instead
+   *
    * Indicates whether team is archived.
    * Apps should use this as a signal to prevent any changes to content associated with archived teams.
    */
   isTeamArchived?: boolean;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.host.name} instead
+   *
    * The name of the host client. Possible values are: Office, Orange, Outlook, Teams
    */
   hostName?: HostName;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.host.clientType} instead
+   *
    * The type of the host client. Possible values are : android, ios, web, desktop, rigel(deprecated, use teamsRoomsWindows instead),
    * surfaceHub, teamsRoomsWindows, teamsRoomsAndroid, teamsPhones, teamsDisplays
    */
   hostClientType?: HostClientType;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.page.frameContext} instead
+   *
    * The context where tab url is loaded (content, task, setting, remove, sidePanel)
    */
   frameContext?: FrameContexts;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.sharePoint} instead
```

Comment:
This property is all lowercase, e.g. "sharepoint"

Reply:
you're right, it's weird to me that the property `sharePointSite` isn't `sharepointSite` 
![image](https://user-images.githubusercontent.com/9360417/178068132-c962f5f4-428d-4e5c-b9aa-5ac42be16017.png)